_field-types/mapping-parameters/eager_global_ordinals.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: Eager global ordinals
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 35
+has_children: false
+has_toc: false
+---
+
+# Eager global ordinals
+
+The `eager_global_ordinals` mapping parameter controls when global ordinals are built for a field. When enabled, global ordinals are computed during index refresh rather than "lazily" during query execution. This can improve performance for operations that rely on global ordinals, for example sorting and aggregations on keyword fields, however it may also increase index refresh times and memory usage.
```

Comment:
```suggestion
The `eager_global_ordinals` mapping parameter controls when global ordinals are built for a field. When enabled, global ordinals are computed during index refresh rather than "lazily" during query execution. This can improve performance for operations that rely on global ordinals, for example, sorting and aggregations on keyword fields. However, it may also increase index refresh times and memory usage.
```